export_options.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "input": ["./logo.svg"],
+    "output": [
+      ["./dist/color_32x32.png", "pad", "32:32"],
+      ["./dist/color_512x512.png", "pad", "512:512"],
+      ["./dist/color_1024x1024.png", "pad", "1024:1024"],
+      ["./dist/black_32x32.png", "pad", "32:32", "@import url(\"http://localhost:4455/black.css\"); svg{}"],
```

Comment:
`svg{}` is included, because the `@import` syntax is not natively recognized as css code.

Reply:
In the package.json we always serve on `4455`, because we need this to be consistent.